package-managers/nuget.md

```diff
@@ -6,10 +6,65 @@ TODO (inspiration: http://docs.myget.org/docs/walkthrough/getting-started-with-n
 
 public feed, private feed with basic auth, pre-authenticated feed URL
 
-### Credential Provider for Visual Studio
+### Find our feed URL
+
+#### Private feed
+
+Private feeds are only accessible to the feed owner, and to other users who were granted access by the feed owner. When we need to connect to a private feed we don't own, we may need to ask the feed owner to invite us to the feed.
+
+<p class="alert alert-success">
+    <strong>Tip:</strong> We can find out how to invite users <a href="../inviting-users-to-a-feed.md">here</a>
+</p>
+
+![Feed details](assets/feed-details.png)
+
+<p class="alert alert-success">
+    <strong>Tip:</strong> We support <a href="#visualstudio">Visual Studio</a>, <a href="#rider">JetBrains Rider</a>, the <a href="#nugetcli">NuGet CLI</a>, <a href="#dotnetcli">dotnet CLI</a> and <a href="#paket">Paket</a>
+</p>
+
+#### Public feed
+
+There are two types of public feeds we can connect to: public feeds we own, or a feed we want to consume from the gallery. The only real difference is where to find the URL of the feed.
+
+MyGet feeds have multiple endpoints, and each one is specific to a particular package type and protocol. Pick the NuGet endpoint URL that matches the protocol version you want to use (e.g. NuGet v3), and use that to configure the feed endpoint in your favorite NuGet client. Let's go over how to connect to these.
+
+#### Gallery feed
+
+After browsing the list of feeds on the [MyGet Gallery](https://myget.org/gallery) (or on the MyGet Enterprise Gallery), we can get its connection details using the **Connect to feed** button. This will open a dialog where we can find the various endpoints for this feed.
+
+![Gallery feed details](assets/gallery-feed.png)

```

Comment:
Maybe add a little bit of text between the two images? Just to make it less dense, visually.